src/main/sensors/gyroanalyse.c

```diff
@@ -130,11 +130,11 @@ const gyroFftData_t *gyroFftData(int axis)
 /*
  * Collect gyro data, to be analysed in gyroDataAnalyseUpdate function
  */
-void gyroDataAnalyse(const gyroDev_t *gyroDev, biquadFilter_t *notchFilterDyn)
+void gyroDataAnalyse(const struct gyroDev_s *gyroDev, biquadFilter_t *notchFilterDyn)
 {
     // if gyro sampling is > 1kHz, accumulate multiple samples
     for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-        fftAcc[axis] += gyroDev->gyroADC[axis];
+        fftAcc[axis] += gyroDev->gyroADCStaticf[axis];
```

Comment:
It may be better to push samples explicitly from gyroUpdateSensor (`gyroDataAnalyseAdd(axis, value)`)